OrbitCore/OrbitModuleTest.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Pdb.h"
```

Comment:
Should this include be at the top?

Reply:
It should. but clang-format does not really understand it...

Reply:
Shouldn't the one at the top be the one with the name of the test? Or in other words, why is OrbitModuleTest a test for Pdb and not OrbitModule?

Reply:
Ops.. I misunderstood the first comment - done.